api/soap/mantisconnect.wsdl

```diff
@@ -45,6 +45,7 @@
    <xsd:element name="account_data" type="tns:AccountData" minOccurs="0"/>
    <xsd:element name="access_level" type="xsd:integer" minOccurs="0"/>
    <xsd:element name="timezone" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="api_token_type" type="xsd:integer" minOccurs="0"/>
```

Comment:
The scenario of crash reporting only deals with one kind of token, the one that only enables reporting.  I would rather not have a surface area for this until we have a good model to define token scopes/types that we feel comfortable exposing via the public API.